src/components/autocompleter/autocompleter.test.jsx

```diff
@@ -45,7 +45,7 @@ describe('src/components/autocompleter/autocompleter', () => {
   describe('validationMessage', () => {
     it('sets the input to be invalid', () => {
       render(<Autocompleter {...requiredProps} validationMessage="not valid" />);
-      expect(screen.getByLabelText('Test label')).toHaveDescription('not valid');
+      expect(screen.getByLabelText('Test label')).toHaveAccessibleDescription('not valid');
```

Comment:
This is needed for the latest testing-library libs. I did a global find and replace.